tmp/LICENSE

```diff
@@ -0,0 +1,674 @@
+                    GNU GENERAL PUBLIC LICENSE
```

Comment:
To chyba skomitowaliście przez przypadek. Polecam dodanie katalogu tmp do .gitignore.